pkg/codegen/dotnet/gen_test.go

```diff
@@ -65,6 +65,7 @@ func TestMakeSafeEnumName(t *testing.T) {
 		{"SystemAssigned, UserAssigned", "SystemAssigned_UserAssigned", false},
 		{"Dev(NoSLA)_Standard_D11_v2", "Dev_NoSLA_Standard_D11_v2", false},
 		{"Standard_E8as_v4+1TB_PS", "Standard_E8as_v4_1TB_PS", false},
+		{"Equals", "EqualsValue", false},
```

Comment:
Is this test case unrelated to the change? (no need to remove it, just for my understanding)

Reply:
Yeah just making sure `Equals` works as expected.